api/v1beta1/tags.go

```diff
@@ -131,6 +131,9 @@ const (
 
 	// MachineNameTagKey is the key for machine name.
 	MachineNameTagKey = "MachineName"
+
+	// NodeRoleTagValue describes the value for the node role.
+	NodeRoleTagValue = "node"
```

Comment:
Let's use this constant when ASG is created as well (currently hardcoded).

Reply:
done